internal/nodes/avi_model_evh_nodes.go

```diff
@@ -826,9 +826,7 @@ func (o *AviObjectGraph) BuildModelGraphForInsecureEVH(routeIgrObj RouteIngressM
 	// Remove the redirect for secure to insecure transition
 	hosts := []string{host}
 	if pathsvcmap.gslbHostHeader != "" {
-		if !utils.HasElem(hosts, pathsvcmap.gslbHostHeader) {
```

Comment:
@swathinsankaran: We have to keep this code base. Reason: As we have removed, gslb to local fqdn check, this will prevent, duplicate fqdn addition. In turn prevent, vip creation failure error. 
Similar for other block.

Reply:
I have added the code back. Thanks.